Makefile

```diff
@@ -22,24 +32,23 @@ tools:
 	go mod tidy
 
 bls:
-	@echo "Compiling bls"
+	@echo "cloning bls"
 	rm -rf $(HERUMI)
-	git clone --recursive git://github.com/herumi/bls.git $(HERUMI)/bls && cd $(HERUMI)/bls && make minimized_static
+	git clone https://github.com/herumi/bls-go-binary.git $(HERUMI)
```

Comment:
MAC users with M1 architecture need to compile BLS manually.

In case we need to use compiled version, we don't need this target.

Reply:
then i guess better to separate release pipe from compiling pipe